src/Bindings/ManualBindings_Network.cpp

```diff
@@ -101,13 +101,12 @@ static int tolua_cNetwork_CreateUDPEndpoint(lua_State * L)
 	}
 
 	// Read the params:
-	int Port;
-	S.GetStackValues(2, Port);
+	UInt16 Port;
 
 	// Check validity:
-	if ((Port < 0) || (Port > 65535))
+	if (S.GetStackValues(2, Port))
```

Comment:
Isn't the condition wrong? "If port-getting succeeded, log warning and bail out"?